### ISSUE TITLE
FitText on bootstrap navbar links

### ISSUE THREAD

**Issue by savgiannis** [davatron5000/FitText.js#166]
Hi, I tried to implement fit text in bootstrap navbar links so the font size is different on xl, lg,  and even on sm screens when it collapses so i will not have to add media queries but it does not work it always keeps the max size. Any help ?
Thanks

**Comment by TrentWalton** [comment]
Hi! You might start by building a reduced test case on a site like codepen so we can see what's up.
Have you seen the readme? https://github.com/davatron5000/FitText.js/blob/master/README.md

**Comment by lwxbr** [comment]
Yeah, links and buttons don't resize using fittext, I tried in this pen:
https://codepen.io/vendramini/pen/PBNqbN

I did some algorithm based in viewport width without change the properties of fittext, hope it might help as it helped me: https://codepen.io/vendramini/pen/jpqVdz
